code/modules/jobs/job_types/roguetown/adventurer/types/antag/pioneer.dm

```diff
@@ -0,0 +1,191 @@
+	allowed_sexes = list(MALE, FEMALE)
+	allowed_races = RACES_ALL_KINDS
+	outfit = /datum/outfit/job/roguetown/bandit/pioneer
+	category_tags = list(CTAG_BANDIT)
+
+	subclass_stats = list(
+		STATKEY_INT = 3,
+		STATKEY_LCK = 1,
+		STATKEY_STR = 1,
+		STATKEY_CON = 1,
+		STATKEY_PER = 1,
+		STATKEY_END = 2 //a lot of stats but no any traits, literally
+	)
+
+/datum/outfit/job/roguetown/bandit/pioneer/pre_equip(mob/living/carbon/human/H)
+	..()
+	if(!istype(H.patron, /datum/patron/inhumen/matthios))
+		var/inputty = input(H, "Would you like to change your patron to Matthios?", "The Transactor calls", "No") as anything in list("Yes", "No")
+		if(inputty == "Yes")
+			to_chat(H, span_warning("My former deity has abandoned me.. Matthios is my new master."))
+			H.set_patron(/datum/patron/inhumen/matthios)
+	belt =	/obj/item/storage/belt/rogue/leather
+	head = /obj/item/clothing/head/roguetown/helmet/kettle
+	pants = /obj/item/clothing/under/roguetown/trou/leather
+	shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/councillor //gambeson but black
+	shoes = /obj/item/clothing/shoes/roguetown/boots/armor //toe safety first
+	mask = /obj/item/clothing/mask/rogue/facemask/steel
+	neck = /obj/item/clothing/neck/roguetown/coif
+	gloves = /obj/item/clothing/gloves/roguetown/angle/grenzelgloves/blacksmith
+	armor = /obj/item/clothing/suit/roguetown/armor/leather/studded
+	backl = /obj/item/storage/backpack/rogue/backpack/
+	backr = /obj/item/rogueweapon/shovel/saperka //40 force +1.5 dmg SMASH but SLOW. If your enemy is a mental retard and stays on one place you delete them
+	beltl = /obj/item/storage/magebag/detpack
+	beltr = /obj/item/flashlight/flare/torch/lantern
+	id = /obj/item/mattcoin
+	backpack_contents = list(
+		/obj/item/lockpickring/mundane = 1,
+		/obj/item/flashlight/flare/torch = 1,
+		/obj/item/restraints/legcuffs/beartrap = 2,
+		/obj/item/flint = 1,
+		/obj/item/rogueore/coal=1,
+		/obj/item/rogueore/iron=1,
+		/obj/item/rogueweapon/hammer/iron = 1,
+		/obj/item/rogueweapon/tongs = 1,
+		/obj/item/rogueweapon/pick/steel
+	)
+
+	H.adjust_skillrank(/datum/skill/combat/bows, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/crossbows, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/axes, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/maces, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/shields,3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/craft/crafting, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/craft/carpentry, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/labor/lumberjacking, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/lockpicking, 3, TRUE)
```

Comment:
If this is going to be a crafting class, they probably shouldn't be getting lockpicking.